src/main/java/be/syntra/devshop/DevshopFront/services/ProductService.java

```diff
@@ -24,12 +24,14 @@
 
     /***
      * to retrieve all products present from the backend
-     * returns a ProductList object
-     * @return ProductList
+     * returns a ProductListDto object
+     * @return ProductListDto
      */
-    ProductList findAllNonArchived();
+    //ProductListDto findAllNonArchived();
```

Comment:
Probably not a good idea to leave commented out code in the codebase

Reply:
removed